rasa/core/channels/console.py

```diff
@@ -60,14 +60,14 @@ def print_bot_output(
             return question
 
     if "text" in message:
-        rasa.shared.utils.cli.print_color(message.get("text"), color=color)
+        rasa.shared.utils.cli.print_color(message["text"], color=color)
```

Comment:
How come we switch to subscript (rather than `get`) here but not in the loops below (for `elements`, `quick_replies`, and `custom`)?

Reply:
Also, in `format_bot_output` (`rasa/core/training/interactive.py`) we check that the values are not `None`. Should we be doing this here, too?

Reply:
Fair point. I think this is due to `elements`, `quick_replies` and `custom` aren't operator issues, as they are wrapped in enumerate calls. I could change it too, or we could say it's out of scope for this issue. I guess it would come up again in another mypy issue. 🤔

Reply:
changed it :)